### PR TITLE
Imp/GitHub theme

### PR DESCRIPTION
**Pull request by alexnm**
* lineHeight exposed on theme
* syntax highlight updated to differentiate between function calls and property access.

### CODE REVIEW

File: sandpack-react/src/components/CodeEditor/utils.ts [hunk 1]
```diff
@@ -90,22 +91,29 @@ export const getSyntaxHighlight = (theme: SandpackTheme): HighlightStyle =>
...
     },
     {
       tag: [tags.literal, tags.inserted],
       ...getSyntaxStyle(theme.syntax.string ?? theme.syntax.static),
     },
-    {
-      tag: tags.propertyName,
-      ...getSyntaxStyle(theme.syntax.property),
-    },
+    // {
```
Comment (danilowoz) [inline sandpack-react/src/components/CodeEditor/utils.ts:112]: Should it be here?

File: sandpack-react/src/components/CodeEditor/utils.ts [hunk 2]
```diff
@@ -90,22 +91,29 @@ export const getSyntaxHighlight = (theme: SandpackTheme): HighlightStyle =>
...
     },
     {
-      tag: tags.typeName,
+      tag: tags.tagName,
       ...getSyntaxStyle(theme.syntax.tag),
     },
     { tag: tags.variableName, ...getSyntaxStyle(theme.syntax.plain) },
     {
+      // Highlight function call
+      tag: tags.function(tags.variableName),
```
Comment (danilowoz) [inline sandpack-react/src/components/CodeEditor/utils.ts:100]: Great!

File: sandpack-react/src/components/CodeEditor/utils.ts [hunk 3]
```diff
@@ -90,22 +91,29 @@ export const getSyntaxHighlight = (theme: SandpackTheme): HighlightStyle =>
       ...getSyntaxStyle(theme.syntax.static),
     },
     {
-      tag: tags.typeName,
+      tag: tags.tagName,
```
Comment (danilowoz) [inline sandpack-react/src/components/CodeEditor/utils.ts:94]: Is this right? Shouldn't we keep the same?

Comment (alexnm) [reply sandpack-react/src/components/CodeEditor/utils.ts:94]: looks to have the same effect, tagName is more suggestive

**Pull request merged**